samcli/commands/check/graph_context.py

```diff
@@ -0,0 +1,22 @@
+"""
+This class will generate the graph based on the template data.
+Connections between nodes will occur here.
+"""
+from .resources.Graph import Graph
+
+
+class GraphContext:
+    def __init__(self, lambda_functions):
+        self.lambda_functions = lambda_functions
+
+    def generate(self):
+        graph = Graph()
+
+        # Find all entry points
+        for function in self.lambda_functions:
+            if function.get_parents() == []:  # No parent resourecs, so this is an entry point
+                graph.add_entry_point(function)
+            else:
```

Comment:
Are we planning to add something else here in the future?
If not, let's remove the else and pass.

Reply:
I planned to add something here, but now it is no longer necessary.